about.html

```diff
@@ -0,0 +1,99 @@
+            </div>
+            <p id="contact-statement">Please contact us via email for any further questions about our car club! info@subaruowners.org</p>
+        </div>
+    </header>
+    <section id="about" class="bg-primary">
+        <div class="container">
+            <div class="row">
+                <div class="col offset-lg-8 text-center mx-auto">
+                    <h2 class="text-white section-heading">Subaru Owners Club 2022 Logo</h2>
+                    <hr class="light my-4">
+                    <p class="text-faded mb-4">The logo of the Subaru Owners Club 2022 was decided through the logo competition from 8th June to 7th July. After online public, voting and selection committees's judging process, the logo from Naresh won the prize. After consultation with Naresh, we optimized the prize winning-logo for this year's summit as follows:</p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section id="services">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12 text-center">
+                    <h2 class="section-heading">See the past club meets</h2>
+                    <hr class="my-4">
+                </div>
+            </div>
+        </div>
+        <div class="container">
+            <div class="row">
+                <div class="col-md-6" id="wheels-event">
+                    <h2>Wheels & Coffee</h2>
+<!--                    <img src="assets/img/thumbnails/gallery_01_11_thumb.jpg" style="box-shadow: 0px 0px rgb(33, 37, 41), 2px 0px;">-->
+                </div>
+                <div class="col-md-6" id="charity-run">
+                    <h2>Isiolo Charity Run</h2>
+<!--                    <img src="assets/img/thumbnails/gallery_01_10_thumb.jpg" style="text-shadow: 2px 0px;box-shadow: 2px 0px;text-align: center;">-->
```

Comment:
- [ ] Please delete the commented code
- [ ] Please center the text on images and add some space between them: 

Your design  | Behance design
------------- | -------------
<img width="762" alt="Screenshot 2022-04-20 at 13 10 48" src="https://user-images.githubusercontent.com/77038610/164193865-6cf4cd25-48d7-415e-96bf-9b5a39b5c666.png"> | <img width="593" alt="Screenshot 2022-04-20 at 13 11 08" src="https://user-images.githubusercontent.com/77038610/164193923-6f6aa2f5-dfd6-47ac-973e-ce333e23c73c.png">

Reply:
Put space in between.
<img width="992" alt="Screenshot 2022-04-21 at 15 08 01" src="https://user-images.githubusercontent.com/40231363/164454885-7c7e3e30-f32b-42cd-a501-2a2f495f1ad8.png">